CUMULUS-1762: respect collection in notifications (#1536)
nasa/cumulus: CHANGELOG.md

## [Unreleased]

### PLEASE NOTE

- **CUMULUS-1762**: the `messageConsumer` for `sns` and `kinesis`-type rules now fetches

>   the collection information from the message. You should ensure that your collection name

Reviewer: ```suggestion
  the collection information from the message. You should ensure that your rule's collection name
```

>   and version line up with what is in the notification for these ingest notifications to

Reviewer: ```suggestion
  and version match what is in the message for these ingest messages to
```

>   process. If no matching rule is found, an error will be thrown and logged in the

Reviewer: ```suggestion
  be processed. If no matching rule is found, an error will be thrown and logged in the
```
